frontend/src/lib/api/config.ts

```diff
@@ -0,0 +1,41 @@
+import { request } from "./request";
+
+export type GetArchiveConfigReturn = {
+  zaaktypesShortProcess: string[]; // List of zaaktypes that should use the short process
+};
+/**
+ * API call to get the archive configuration
+ * @returns {Promise<GetArchiveConfigReturn>} The archive configuration
+ * @throws {Error} If the request fails
+ * @example
+ * const archiveConfig = await getArchiveConfig();
+ * console.log(archiveConfig);
+ */
+export async function getArchiveConfig(): Promise<GetArchiveConfigReturn> {
+  const response = await request("GET", "/archive-config");
+  const promise: Promise<GetArchiveConfigReturn> = response.json();
+  return promise;
+}
+
+export type PatchArchiveConfigPayload = GetArchiveConfigReturn;
```

Comment:
The payload can be simply an `ArchiveConfiguration` (if the previous comment is applied), I don't see a reason in loose coupling void here?